modules/ROOT/pages/troubleshoot-access-management.adoc

```diff
@@ -0,0 +1,41 @@
+= Troubleshoot Access Management
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+These are some tips about what to do if you encounter these issues. 
+
+== Problems with Accessing Anypoint Platform
+
+=== Sign in Credentials are not recognized
+
+If your credentials aren't recognized when you try to sign into Anypoint Platform, try the following:
+
+* Check that you are using the correct URL to sign into your organization. +
+If your organization uses an external identity management system, your login URL is *anypoint.mulesoft.com/accounts/login/_yourOrganizationName_*.
+* Verify that you are using the correct username and password for the account
+* If your organization isn't using an external identity management system and your sign in credentials aren't working, reset your password or ask your Organization Administrator if your account is still active.
+
+=== Access Management invitation link returns an error
+
+If you click the link in your invitation and an error is returned, your invitation may have expired or been rescinded. Please contact your Organization Administrator for a new invitation.
+
+== OpenID Connect Client Management
+
+=== Error received when you try to update a client application
+
+If you receive an `There was an error while talking to CoreServices` when you try to update a client application under OpenID Connect management, this is because updates of clients created through this integration are not currently supported.
```

Comment:
I prefer fixing the product than documenting a bad interaction with the user explicitly. If we are not going to fix it at least is documented now.

Reply:
Pablo, this is a limitation of the OIDC dynamic client registration spec. Update/delete is not defined (but many implementations support it)

Reply:
I was referring to the message which is the same for any communication error if I'm not wrong. I know it needs collaboration from API Manager. The user doesn't know what kind of error it is and even what CoreServices is?